operator/internal/manifests/internal/config/options.go

```diff
@@ -31,15 +34,46 @@ type Address struct {
 	Port int
 }
 
-// ObjectStorage for storage config.
-type ObjectStorage struct {
+// AzureObjectStorage for Azure storage config
+type AzureObjectStorage struct {
```

Comment:
I see some duplication between the structs defined here and in `manifests/options.go`. Should we try to consolidate these types so that they are only defined once?

Reply:
I had thought about it considering the overlap, but I was sure as there was this similarity already between the structures. Though perhaps that might because they were just simple objects...